src/lib/kdf/sp800_56a/sp800_56a.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+* KDF defined in NIST SP 800-56a (Approved Alternative 1)
+*
+* (C) 2017 Ribose Inc.
+* Written by Krzysztof Kwiatkowski.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <botan/sp800_56a.h>
+#include <botan/hmac.h>
+#include <botan/scan_name.h>
+#include <botan/exceptn.h>
+
+namespace Botan {
+
+namespace {
+static const uint64_t MAX_REPS = (2ULL << 32);
```

Comment:
static in a cpp file and anonymous namespace is basically the same thing for variables, so the "static" keyword should be removed here;

Regarding naming: The value you define here is not a maximum but a strict upper bound, right?

Regarding value: this is shifting binary 11 to the left, which does not end up as 2^32